docs/feature-gates.md

```diff
@@ -138,3 +139,28 @@ foo           bar                   1          12        1221    2020-09-07T13:2
 #### Requirements for this Feature
 
 None
+
+### NodePortLocal
+
+`NodePortLocal` is a feature that runs as part of the Antrea Agent, through which
+each port of a Pod can be reached from external network using a port in the Node
+on which the Pod is running. In addition to enabling NodePortLocal feature gate,
+the value of `nplPortRange` must be set in Antrea Agent configuration through ConfigMap.
+Ports from a Node would only be allocated from the range of ports specified in `nplPortRange`.
+
+Pods can be selected for `NodePortLocal` by tagging a Service with Annotation:
+`nodeportlocal.antrea.io/enabled: "true"`. Consequently, `NodePortLocal` is enabled
+for all the Pods which are selected by the Service through a selector, and the ports of these
+Pods would be reachable through Node port.
```

Comment:
suggestion: "would be reachable through Node ports allocated from the `nplPortRange`."

Reply:
Updated.